bindings/gen_bindings.sh

```
@@ -54,4 +54,4 @@ abigen \
 # Place import "github.com/ethereum/go-ethereum" to "github.com/morph-l2/go-ethereum"
 TO_REPLACED_STRING="github.com/ethereum/go-ethereum"
 NEW_STRING="github.com/morph-l2/go-ethereum"
-sed -i "" -e "s#${TO_REPLACED_STRING}#${NEW_STRING}#g" ./${PACKAGE}/${TYPE_LOWER}.go
+sed -i.bak -e "s#${TO_REPLACED_STRING}#${NEW_STRING}#g" ./${PACKAGE}/${TYPE_LOWER}.go && rm -f ./${PACKAGE}/${TYPE_LOWER}.go.bak
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)